dashboard/app/main.go

```diff
@@ -2431,26 +2424,30 @@ func invalidateJobLink(c context.Context, job *Job, jobKey *db.Key, restart bool
 	return "/admin?" + params.Encode()
 }
 
-func formatLogLine(line string) string {
+func appendLogLine(lines *[]string, line string) {
+	// Something related to package updates on GCE machines.
+	// They are happening in hundreds every day.
+	if strings.Contains(line, "packages.GetPackageUpdates") ||
+		// Let somebody else monitor datastore bugs.
+		strings.Contains(line, "datastore_v3: INTERNAL_ERROR") {
```

Comment:
Please mention https://github.com/google/syzkaller/issues/6069 .

Context: higher level datastore lib is able to automatically retry after 5xx errors. We struggle from 5xx errors because use low-level lib.